src/main/java/org/sonatype/gradle/plugins/scan/ossindex/OssIndexAuditTask.java

```diff
@@ -0,0 +1,215 @@
+/*
+ * Copyright (c) 2020-present Sonatype, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.sonatype.gradle.plugins.scan.ossindex;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.net.UnknownHostException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Map.Entry;
+import java.util.stream.Collectors;
+
+import org.sonatype.goodies.packageurl.PackageUrl;
+import org.sonatype.goodies.packageurl.PackageUrlBuilder;
+import org.sonatype.gradle.plugins.scan.common.DependenciesFinder;
+import org.sonatype.ossindex.service.api.componentreport.ComponentReport;
+import org.sonatype.ossindex.service.api.componentreport.ComponentReportVulnerability;
+import org.sonatype.ossindex.service.client.OssindexClient;
+import org.sonatype.ossindex.service.client.OssindexClientConfiguration;
+import org.sonatype.ossindex.service.client.internal.OssindexClientImpl;
+import org.sonatype.ossindex.service.client.marshal.GsonMarshaller;
+import org.sonatype.ossindex.service.client.marshal.Marshaller;
+import org.sonatype.ossindex.service.client.transport.Transport;
+import org.sonatype.ossindex.service.client.transport.Transport.TransportException;
+
+import org.apache.commons.lang3.StringUtils;
+import org.gradle.api.DefaultTask;
+import org.gradle.api.GradleException;
+import org.gradle.api.artifacts.ResolvedArtifact;
+import org.gradle.api.tasks.Input;
+import org.gradle.api.tasks.TaskAction;
+import org.gradle.internal.impldep.com.google.common.annotations.VisibleForTesting;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class OssIndexAuditTask
+    extends DefaultTask
+{
+  private final Logger log = LoggerFactory.getLogger(OssIndexAuditTask.class);
+
+  private final OssIndexPluginExtension extension;
+
+  private final DependenciesFinder dependenciesFinder;
+
+  public OssIndexAuditTask() {
+    extension = getProject().getExtensions().getByType(OssIndexPluginExtension.class);
+    dependenciesFinder = new DependenciesFinder();
+  }
+
+  @TaskAction
+  public void audit() {
+    boolean hasVulnerabilities = false;
```

Comment:
Minor: Redundant initializer. No need to assign false, it is the default

Reply:
> Minor: Redundant initializer. No need to assign false, it is the default

Done: 61e8b7fec0f3fc531e9813f711419a7718f60dd9